README.md

```diff
@@ -60,7 +60,7 @@ the extremely long folder name.
 
 ## Publishing
 
-Publishing happens through the CI/CD pipeline. See [package.json](.travis.yml)
+Publishing happens through the CI/CD pipeline. See [package.json](.github/workflows/ci-cd.yml)
```

Comment:
We should have an explanatino instead of just saying CI/CD. Two sections, creating an `@next` release and creating an `@latest`